xtask/src/commands/changeset/mod.rs

```diff
@@ -0,0 +1,499 @@
+    /// themselves and also for the `--class` flag for the CLI.
+    fn as_short_name(&self) -> &'static str {
+        match self {
+            Classification::Breaking => "breaking",
+            Classification::Feature => "feat",
+            Classification::Fix => "fix",
+            Classification::Configuration => "config",
+            Classification::Maintenance => "maint",
+            Classification::Documentation => "docs",
+            Classification::Experimental => "exp",
+        }
+    }
+
+    /// This definition of how to iterate over classifications is responsible
+    /// for the ordering that eventually appears in the emitted CHANGELOG. It's
+    /// also the order options are displayed in the TUI.
+    pub fn iterator() -> Iter<'static, Classification> {
+        static CLASSIFICATIONS: [Classification; 7] = [
+            Classification::Breaking,
+            Classification::Feature,
+            Classification::Fix,
+            Classification::Configuration,
+            Classification::Maintenance,
+            Classification::Documentation,
+            Classification::Experimental,
+        ];
+        CLASSIFICATIONS.iter()
+    }
+}
+
+type ParseError = &'static str;
+impl FromStr for Classification {
+    type Err = ParseError;
+    fn from_str(classification: &str) -> Result<Self, Self::Err> {
+        match classification {
+            "break" => Ok(Classification::Breaking),
+            "feat" => Ok(Classification::Feature),
+            "fix" => Ok(Classification::Fix),
+            "config" => Ok(Classification::Configuration),
+            "maint" => Ok(Classification::Maintenance),
+            "docs" => Ok(Classification::Documentation),
+            "exp" => Ok(Classification::Experimental),
+            &_ => Err("unknown classification"),
+        }
+    }
+}
+
+impl fmt::Display for Classification {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let pretty = match self {
+            Classification::Breaking => "❗ BREAKING ❗",
+            Classification::Feature => "🚀 Features",
+            Classification::Fix => "🐛 Fixes",
+            Classification::Configuration => "📃 Configuration",
+            Classification::Maintenance => "🛠 Maintenance",
+            Classification::Documentation => "📚 Documentation",
+            Classification::Experimental => "🧪 Experimental",
+        };
+        write!(f, "{}", pretty)
+    }
+}
+
+#[derive(Debug, StructOpt)]
+pub struct Create {
+    /// Use the current branch as the file name
+    #[structopt(short = "b", long = "--with-branch-name")]
+    with_branch_name: bool,
+
+    /// The classification of the changeset
+    #[structopt(short = "c", long = "--class")]
+    classification: Option<Classification>,
+}
+
+async fn github_graphql_post_request(
+    token: &str,
+    url: &str,
+    request_body: &graphql_client::QueryBody<
+        matching_pull_request::matching_pull_request::Variables,
```

Comment:
`Variables` is already imported, so this can be shorter

Reply:
Thanks! Fixed in bea2964f